### DISCUSSION TITLE
Accessing Nextcloud from my home network

### DISCUSSION THREAD

**Discussion by clareclever** [nextcloud/all-in-one#1291] [Questions] [answered]
Hello, 

I am having some troubles accessing my AIO Nextcloud instance from my home network. I can access the admin page just fine. https://my.public.url.org:8443/containers. However, when I hit the button on that page to take me to the normal site, it fails. I can access other sites/services using my public domain on separate ports, Plex for example. It's only Nextcloud that won't work. I have all of the correct rules setup in my router and firewall. I have port 80 and 443 open, which is what it should use when I go to access the base site. Is there anything else I need to setup inside of Nextcloud to grant me access to the base site on my home network?

Please let me know,
Thanks!

**Comment by szaimen** [comment, marked as answer]
Lets keep the discussion in the forum: https://help.nextcloud.com/t/accessing-nextcloud-on-my-home-network/147657
